client/reader/start/main.jsx

```diff
@@ -55,8 +57,16 @@ const Start = React.createClass( {
 		page.redirect( '/' );
 	},
 
+	renderLoadingPlaceholders() {
+		const count = 4;
```

Comment:
I meant [config](https://github.com/Automattic/wp-calypso/blob/master/config). Back in the cloudup days we'd use config for things like this that are essentially magic numbers. `const numberOfPlaceholders = config('reader_number_of_placeholders')`.

Though taking a look at the current config is doesn't seem like we're using that for things like this, so probably nvm 🍤